python/src/sdf/pyLink.cc

```diff
@@ -0,0 +1,205 @@
+         "is relative to the parent model.")
+    .def("set_pose_relative_to",
+         &sdf::Link::SetPoseRelativeTo,
+         "Set the name of the coordinate frame relative to which this "
+         "object's pose is expressed. An empty value indicates that the frame "
+         "is relative to the parent model.")
+    .def("semantic_pose",
+         &sdf::Link::SemanticPose,
+         "Get SemanticPose object of this object to aid in resolving "
+         "poses.")
+    .def("Inertial",
+         &sdf::Link::Inertial,
+         "Get the inertial value for this link.")
+    .def("SetInertial",
+         &sdf::Link::SetInertial,
+         "Set the inertial value for this link.")
+    .def("raw_pose", &sdf::Link::RawPose,
+         "Get the pose of the link object. This is the pose of the "
+         "link as specified in SDF")
+    .def("set_raw_pose", &sdf::Link::SetRawPose,
+         "Set the pose of the link object.")
+    .def("pose_relative_to", &sdf::Link::PoseRelativeTo,
+         "Get the name of the coordinate frame relative to which this "
+         "object's pose is expressed. An empty value indicates that the frame "
+         "is relative to the parent model.")
+    .def("set_pose_relative_to", &sdf::Link::SetPoseRelativeTo,
+         "Set the name of the coordinate frame relative to which this "
+         "object's pose is expressed. An empty value indicates that the frame "
+         "is relative to the parent model.")
+    .def("semantic_pose", &sdf::Link::SemanticPose,
+         "Get SemanticPose object of this object to aid in resolving "
+         "poses.")
+    .def("enable_wind",
+         &sdf::Link::EnableWind,
+         "Check if this link should be subject to wind. "
+         "If true, this link should be affected by wind.")
+    .def("set_enable_wind",
+         &sdf::Link::SetEnableWind,
+         "Set whether this link should be subject to wind.")
+    .def("add_collision",
+         &sdf::Link::AddCollision,
+         "Add a collision to the link.")
+    .def("add_visual",
+         &sdf::Link::AddVisual,
+         "Add a visual to the link.")
+    // .def("AddLight",
+    //      &sdf::Link::AddLight,
+    //      "Add a visual to the link.")
+    // .def("AddSensor",
+    //      &sdf::Link::AddSensor,
+    //      "Add a sensor to the link.")
+    // .def("AddParticleEmitter",
+    //      &sdf::Link::AddParticleEmitter,
+    //      "Add a particle emitter to the link.")
+    .def("clear_collisions",
+         &sdf::Link::ClearCollisions,
+         "Remove all collisions")
+    .def("clear_visuals",
+         &sdf::Link::ClearVisuals,
+         "Remove all visuals")
+    .def("clear_lights",
+         &sdf::Link::ClearLights,
+         "Remove all lights")
+    .def("clear_sensors",
+         &sdf::Link::ClearSensors,
+         "Remove all sensors")
+    .def("clear_particle_emitters",
+         &sdf::Link::ClearParticleEmitters,
+         "Remove all particle emitters")
```

Comment:
We should leave these out until lights, sensors and particle emitters are 
implemented.